src/validations/index.js

```diff
@@ -1,4 +1,11 @@
+var i18n = require('i18next-client');
+i18n.init({
+  lng: 'en',
```

Comment:
Can we grab this from the browser? With a fallback to English for languages we don’t support?